src/extensions/struct-conn/index.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) 2018-2023 mol* contributors, licensed under MIT, See LICENSE file for more info.
+ *
+ * @author Adam Midlik <midlik@gmail.com>
+ */
+
+import { PluginContext } from '../../mol-plugin/context';
+
+
+export namespace StructConnExtensionFunctions {
```

Comment:
We are trying to move away from `namespace` usage. Please export this for example as a `export const StructConnExtensions`.